src/shared/components/organisms/Settings/molecules/SettingTokenCreate.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useCallback, ChangeEvent } from 'react'
```

Comment:
`TokenCreate` is not a `shared`
component. Local space does not have token creation, only cloud. 
Keep it in the cloud folder.